README.md

```diff
@@ -174,12 +175,13 @@ A valid configuration file should look like this:
 
 The following table describes the configuration properties defined above.
 
-| Setting                 | Description                        | Default        |
-|-------------------------|------------------------------------|----------------|
-| bridge.id               | ID of the bridge                   | my-bridge      |
-| mqtt.host               | Host address of the MQTT server    | localhost      |
-| mqtt.port               | Port number of the MQTT server     | 1883           |
-| kafka.bootstrap.servers | Bootstrap servers for Apache Kafka | localhost:9092 |
+| Setting                        | Description                                                                                      | Default          |
+|--------------------------------|--------------------------------------------------------------------------------------------------|------------------|
+| bridge.id                      | ID of the bridge                                                                                 | my-bridge        |
+| bridge.default.topic[Optional] | Default Kafka topic for the bridge. If not set, the bridge will use the `messages_default` topic | messages_default |
```

Comment:
```suggestion
| bridge.topic.default | Default Kafka topic for the bridge. If not set, the bridge will use the `messages_default` topic | messages_default |
```

you should explain better that this topic is where messages are forwarded when there is no match with the topic mapping rules.

Reply:
It is explained early in the docs. However,  I found  it useful to explain here as well. Thanks for the suggestions